src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setClients(String)}
+     */
+    String[] clients() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMatchers(String)}
+     */
+    String[] matchers() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMultiProfile(boolean)}
+     */
+    boolean multiProfile() default false;
```

Comment:
why is that? It only makes sense in the case, again, of configuration files for servlet that are not typed by a compiler. In our case, this situation does not arise, so a boolean is enough.

Also, surprisingly, annotations do not accept null default value!